### PR TITLE
this is enough for 2day

### PR DESCRIPTION
**Pull request by shedaniel**
plz accept

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ plugins {
 	id 'maven-publish'
 }
 
-def minecraft_version = "1.2.5"
+def minecraft_version = "b1.7.3"
```
Comment (valoeghese) [inline build.gradle]: Nice

File: mappings/net/minecraft/gui/DrawableHelper.mapping [hunk 2]
```diff
@@ -0,0 +1,15 @@
+CLASS ub net/minecraft/gui/DrawableHelper
```
Comment (valoeghese) [inline mappings/net/minecraft/gui/DrawableHelper.mapping]: epic

File: mappings/net/minecraft/gui/Screen.mapping [hunk 3]
```diff
@@ -0,0 +1,24 @@
+CLASS da net/minecraft/gui/Screen
```
Comment (valoeghese) [inline mappings/net/minecraft/gui/Screen.mapping]: I assume this also applies to non GUI screens then @shedaniel ?

Comment (valoeghese) [reply mappings/net/minecraft/gui/Screen.mapping]: Nvm it does. I just saw "downloading terrain screen"

File: mappings/net/minecraft/sortme/ItemStack.mapping [hunk 4]
```diff
@@ -0,0 +1,5 @@
+CLASS iz net/minecraft/sortme/ItemStack
```
Comment (valoeghese) [inline mappings/net/minecraft/sortme/ItemStack.mapping]: @shedaniel this needs to be `Item`! We can't have any item stacks here in cursed land.

**Comment by valoeghese** [comment]
Requested change: `ItemStack` -> `Item`

File: mappings/net/minecraft/gui/DrawableHelper.mapping [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+	FIELD k zOffset F
+	METHOD a fillGredient (IIIIII)V
+	METHOD a drawTextWithShadowCentered (Lsj;Ljava/lang/String;III)V
+		ARG 1 textManager
+		ARG 2 text
+		ARG 3 x
+		ARG 4 y
+		ARG 5 color
```
Comment (valoeghese) [inline mappings/net/minecraft/gui/DrawableHelper.mapping]: `colour`

File: mappings/net/minecraft/gui/DrawableHelper.mapping [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+	METHOD b drawTextWithShadow (Lsj;Ljava/lang/String;III)V
+		ARG 1 textManager
+		ARG 2 text
+		ARG 3 x
+		ARG 4 y
+		ARG 5 color
```
Comment (valoeghese) [inline mappings/net/minecraft/gui/DrawableHelper.mapping]: `colour`

File: mappings/net/minecraft/gui/DrawableHelper.mapping [hunk 7]
```diff
@@ -0,0 +1,15 @@
+CLASS ub net/minecraft/gui/DrawableHelper
+	FIELD k zOffset F
+	METHOD a fillGredient (IIIIII)V
+	METHOD a drawTextWithShadowCentered (Lsj;Ljava/lang/String;III)V
```
Comment (valoeghese) [inline mappings/net/minecraft/gui/DrawableHelper.mapping]: `drawTextWithShadowCentred` @shedaniel

File: mappings/net/minecraft/item/ItemType.mapping [hunk 8]
```diff
@@ -1,11 +1,55 @@
...
+	FIELD aN EGG Lgm;
+	FIELD aO COMPASS Lgm;
+	FIELD aP FISHING_ROD Lgm;
+	FIELD aQ CLOCK Lgm;
+	FIELD aR YELLOW_DUST Lgm;
+	FIELD aS FISH_RAW Lgm;
+	FIELD aT FISH_COOKED Lgm;
+	FIELD aU DYE_POWDER Lgm;
+	FIELD aV BONE Lgm;
+	FIELD aW SUGER Lgm;
```
Comment (valoeghese) [inline mappings/net/minecraft/item/ItemType.mapping]: `SUGAR`

File: mappings/net/minecraft/gui/DrawableHelper.mapping (hunk 7 above)
Comment (valoeghese) [reply mappings/net/minecraft/gui/DrawableHelper.mapping]: (center -> centre)

Also `fillGredient` -> `fillGradient` unless that's intended

**Comment by liach** [comment]
Why is this british

**Comment by valoeghese** [comment]
@liach Because:
1. I'm from New Zealand
2. The mappings are cursed

**Comment by shedaniel** [comment]
yes accept cursed

**Comment by valoeghese** [comment]
thank

**Pull request merged**